configuration/configuration.go

```diff
@@ -609,6 +610,21 @@ func (c *ConfigurationData) GetAuthServiceURL() string {
 	}
 }
 
+// GetUIURL returns UI URL
+func (c *ConfigurationData) GetUIURL() string {
+	if c.v.IsSet(varUIURL) {
+		return c.v.GetString(varUIURL)
+	}
+	switch c.GetEnvironment() {
+	case prodEnvironment:
+		return "https://openshift.io"
```

Comment:
can't we have a better way for handling URLs ? Can't we have an endpoint in WIT that would return the URL for the space, instead ?

Reply:
currently it's not. you won't get ui url for space.i.e. `http://openshift.io/dpawar/myspace`. You will get `http://wit/spaces/:spaceid`